src/table.tsx

```diff
@@ -115,6 +116,7 @@ export const Table = React.forwardRef(
             columnWidth,
             hideHeaders,
             scrollOverResponsiveLayout,
+            rowHeaderIndex = 0,
```

Comment:
Does this mean we'll always have a `<th scope="row">`? I think we should support the case of not having one, and I'd say it should be the default case

Reply:
Discussed offline. We decided to remove default header index and don't show a title in collapsed-rows mode if row's header index has not been specified